<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrmTasksList</class>
 <widget class="QWidget" name="CFrmTasksList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <property name="windowIcon">
   <iconset theme="task">
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout"/>
  <action name="actionLoad_L">
   <property name="icon">
    <iconset theme="document-open">
    </iconset>
   </property>
   <property name="text">
    <string>Load(&amp;L)</string>
   </property>
   <property name="toolTip">
    <string>Load tasks</string>
   </property>
   <property name="statusTip">
    <string>Load tasks</string>
   </property>
   <property name="whatsThis">
    <string>Load tasks</string>
   </property>
  </action>
  <action name="actionSaveAs_S">
   <property name="icon">
    <iconset theme="document-save-as">
    </iconset>
   </property>
   <property name="text">
    <string>Save as(&amp;S) ......</string>
   </property>
   <property name="iconText">
    <string>Save tasks as ......</string>
   </property>
   <property name="toolTip">
    <string>Save tasks as ......</string>
   </property>
   <property name="statusTip">
    <string>Save tasks as ......</string>
   </property>
   <property name="whatsThis">
    <string>Save tasks as ......</string>
   </property>
  </action>
  <action name="actionNew_N">
   <property name="icon">
    <iconset theme="add">
    </iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>New tasks</string>
   </property>
   <property name="statusTip">
    <string>New tasks</string>
   </property>
   <property name="whatsThis">
    <string>New tasks</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionRemove_R">
   <property name="icon">
    <iconset theme="edit-delete">
    </iconset>
   </property>
   <property name="text">
    <string>Remove(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>Remove tasks</string>
   </property>
   <property name="statusTip">
    <string>Remove tasks</string>
   </property>
   <property name="whatsThis">
    <string>Remove tasks</string>
   </property>
  </action>
  <action name="actionRefresh_F">
   <property name="icon">
    <iconset theme="view-refresh">
    </iconset>
   </property>
   <property name="text">
    <string>Refresh(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string>Refresh</string>
   </property>
   <property name="whatsThis">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource/ResourceTasks.qrc"/>
 </resources>
 <connections/>
</ui>
